src/Gom/main.json

```diff
@@ -0,0 +1,6 @@
+{
+  "category": "feedback",
+  "description": "demo showcases inserting and retrieving items in a SQLite database using GOM",
```

Comment:
Let's put this into platform and pick a nice, short description. Maybe something like "Search, load and store GObjects in a SQLite database".

Reply:
Done!